lib/food_order.rb

```diff
@@ -0,0 +1,30 @@
+require_relative './takeaway_menu'
+
+class FoodOrder
+
+  attr_reader :food_order
+
+  def initialize
+    @food_order = []
+    @menu = Menu.new
+  end
+
+  def add_item(food, number)
+    raise "This menu item does not exist" if @menu.menu[food].nil?
+
+    menu_item = @menu.menu.select { |key, _value| key == food }
+    number.times { @food_order << menu_item }
+    order_sum
+  end
+
+  def order_sum
+    order_total = 0
+    @food_order.each do |order_item|
+      order_item.each do |_key, value|
+        order_total += value
+      end
+    end
+    puts "The cost of your order is £%.2f" % order_total
```

Comment:
It'd be better to return the value, rather than printing a message including it — this way the method `order_sum` only deals with data, and only returns a useful result (the total sum), and printing it could be done in some other place in the code — remember separation of concerns also mean that one class dealing with data such as `FoodOrder` shouldn't be also responsible for printing/displaying something